### ISSUE TITLE
this isn't going to work as is since we haven't defined core_samp

### ISSUE THREAD

**Issue by BenWilliams-NOAA** [BenWilliams-NOAA/surveyISS#22]
https://github.com/BenWilliams-NOAA/surveyISS/blob/683fc6dff0487c2a46ad50a5470a9475a50b5391/R/reduce_samples.R#L14

`core_samp()` should either be defined in the utils.R script or as `swo::core_samp`

**Comment by pete-hulson** [comment]
good catch - added in what the core_samp function did here, also realized we needed to ungroup the .redux_samp. still haven't tested this yet, so will leave this open till i get around to it
